autogaita/autogaita_dlc_gui.py

```diff
@@ -466,7 +449,7 @@ def build_cfg_window(root, cfg, root_dimensions):
         command=lambda: build_column_info_window(root, cfg, root_dimensions),
     )
     column_info_button.grid(
-        row=17, column=0, rowspan=2, sticky="nsew", padx=10, pady=(10, 5)
+        row=15, column=0, rowspan=2, sticky="nsew", padx=10, pady=(10, 5)
```

Comment:
I did not make these changes, they are a result of me using an older main branch. Therefore is assume the "old" line is the correct one, right?